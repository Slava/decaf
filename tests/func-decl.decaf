
void MyFunction (int a, bool b) {}

