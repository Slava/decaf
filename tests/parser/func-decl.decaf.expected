Program
  Function void: MyFunction: <int a>, <bool b>

